packages/apollo-server-core/src/plugin/landingPage/default/index.ts

```diff
@@ -16,11 +16,24 @@ export interface ApolloServerPluginLandingPageDefaultBaseOptions {
   footer?: boolean;
   /**
    * Folks can configure their landing page to link to Studio Explorer with a
-   * document, variables and headers loaded in the UI.
+   * document loaded in the UI.
    */
   document?: string;
+  /**
+   * Folks can configure their landing page to link to Studio Explorer with
```

Comment:
Now that it's written four times I'm starting to wonder if "folks can" is the best way to start a doc string.  I kinda love it and I kinda hate it. What do you think?